share/models/creative.py

```diff
@@ -39,7 +39,7 @@ def normalize(self, node, graph):
     class Disambiguation:
         any = ('identifiers',)
 
-    class Meta:
+    class Meta(ShareObject.Meta):
```

Comment:
What is this doing? I haven't seen anything in the release notes that would warrant doing this.

Does the `abstract = True` mess things up at all?